crates/bevy_core_pipeline/src/core_3d/mod.rs

```diff
@@ -356,9 +383,9 @@ pub fn prepare_core_3d_depth_textures(
                     sample_count: msaa.samples(),
                     dimension: TextureDimension::D2,
                     // PERF: vulkan docs recommend using 24 bit depth for better performance
-                    format: TextureFormat::Depth32Float,
+                    format: CORE_3D_DEPTH_FORMAT,
                     usage,
-                    view_formats: &[],
+                    view_formats: &[CORE_3D_DEPTH_FORMAT],
```

Comment:
I don't think we need to declare a view_format here, since it's the same format?

Unless it's some depth+stencil weirdness?

Reply:
I think you're right. It's not needed.